### DISCUSSION TITLE
is it possible to use `      options: {xtream_skip_vod: true, xtream_skip_series: true}` on a per user and/or taget basis?

### DISCUSSION THREAD

**Discussion by aniel300** [euzu/tuliprox#125] [General]
is it possible to use `      options: {xtream_skip_vod: true, xtream_skip_series: true}` on a per user and/or taget basis? it seems this optin only works for the input but not for the output or user, is that correct?

**Comment by euzu** [comment]
This option means do not fetch vod or series from source/input.

If you want to skip  for a target use NOT-Filter with `Type = vod`  or `Type = series`

**Reply by aniel300** [reply]
ok let me try that

**Reply by aniel300** [reply]
still need to try this (been procrastinating that is why i haven't close it)

**Comment by euzu** [comment]
closed due inactivity
